bot/exts/help_channels/_cog.py

```diff
@@ -1,656 +1,74 @@
-        timestamp = arrow.Arrow.fromdatetime(message.created_at).timestamp()
-
-        await _caches.claim_times.set(message.channel.id, timestamp)
-        await _caches.claimant_last_message_times.set(message.channel.id, timestamp)
-        # Delete to indicate that the help session has yet to receive an answer.
-        await _caches.non_claimant_last_message_times.delete(message.channel.id)
-
-        # Removing the help channel from the dynamic message, and editing/sending that message.
-        self.available_help_channels.remove(message.channel)
-
-        # Not awaited because it may indefinitely hold the lock while waiting for a channel.
-        scheduling.create_task(self.move_to_available(), name=f"help_claim_{message.id}")
-
-    def create_channel_queue(self) -> asyncio.Queue:
-        """
-        Return a queue of dormant channels to use for getting the next available channel.
-
-        The channels are added to the queue in a random order.
-        """
-        log.trace("Creating the channel queue.")
-
-        channels = list(_channel.get_category_channels(self.dormant_category))
-        random.shuffle(channels)
-
-        log.trace("Populating the channel queue with channels.")
-        queue = asyncio.Queue()
-        for channel in channels:
-            queue.put_nowait(channel)
-
-        return queue
-
-    async def create_dormant(self) -> t.Optional[discord.TextChannel]:
-        """
-        Create and return a new channel in the Dormant category.
-
-        The new channel will sync its permission overwrites with the category.
-
-        Return None if no more channel names are available.
-        """
-        log.trace("Getting a name for a new dormant channel.")
-
-        try:
-            name = self.name_queue.popleft()
-        except IndexError:
-            log.debug("No more names available for new dormant channels.")
-            return None
-
-        log.debug(f"Creating a new dormant channel named {name}.")
-        return await self.dormant_category.create_text_channel(name, topic=HELP_CHANNEL_TOPIC)
+        self.help_forum_channel_id = constants.Channels.help_system_forum
 
     async def close_check(self, ctx: commands.Context) -> bool:
-        """Return True if the channel is in use and the user is the claimant or has a whitelisted role."""
-        if ctx.channel.category != self.in_use_category:
-            log.debug(f"{ctx.author} invoked command 'close' outside an in-use help channel")
+        """Return True if the channel is a help post, and the user is the claimant or has a whitelisted role."""
+        if not _channel.is_help_forum_post(ctx.channel):
             return False
 
-        if await _caches.claimants.get(ctx.channel.id) == ctx.author.id:
+        if ctx.author.id == ctx.channel.owner_id:
             log.trace(f"{ctx.author} is the help channel claimant, passing the check for dormant.")
             self.bot.stats.incr("help.dormant_invoke.claimant")
```

Comment:
Should we unify the top key for stats? Some are under `help_forum` and some under `help`.

Reply:
The only reason I kept this one the same is to keep the previous stats together with the new stats

Reply:
Fair enough, that works for me.